docs/user-guide/rules/performance-budget.md

```diff
@@ -0,0 +1,147 @@
+# Performance budget (`performance-budget`)
+
+> A web performance budget is a group of limits to certain values that affect
+> site performance that should not be exceeded in the design and development of
+> any web project. This could be the total size of a page, size of images you
+> are uploading, or even the number of HTTP requests that your webpage
+> generates.
+
+https://www.keycdn.com/blog/web-performance-budget/
+
+## Why is this important?
+
+As of January 2018, the average size of a website is 3,545kB:
+
+![image](https://chart.googleapis.com/chart?chs=400x225&cht=p&chco=007099&chd=t:1818,70,98,504,120,851,27&chds=0,1818&chdlp=b&chdl=total%203545%20kB&chl=Images+-+1818+kB%7CHTML+-+70+kB%7CStylesheets+-+98+kB%7CScripts+-+504+kB%7CFonts+-+120+kB%7CVideo+-+851+kB%7COther+-+27+kB&chma=|5&chtt=Average+Bytes+per+Page+by+Content+Type)
+
+Although the global average connection is 7.2Mb/s (check [_Akamai's state of
+the Internet 2017_][state of the internet]), "no bit is faster than one that is
+not sent" (quote by [Ilya Grigorik][faster bit]). Web developers need to be
+mindful not only about the size of their sites, but also the number of
+requests, different domains, third party scripts, etc. The time required by a
+browser to download a 200kB file is not the same than 20 files of 10kB.
+
+## What does the rule check?
+
+This rule calculates how long it will take to download all the resources loaded
+initially by the website under a `3G Fast` network. If the load time is
```

Comment:
Maybe add something in the lines of: `3G Fast` network (but that can be changed, see "Can the rule be configured?" section.